skt/console.py

```diff
@@ -11,6 +11,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+"""Class for managing ConsoleLog."""
```

Comment:
This commit definitely needs to be split. If nothing else, then into one adding the module docstrings, one adding the docstrings for the `merge_*` methods and another one for the `__get_remote_name`, because all of them are unrelated.

Reply:
I agree.